app/models/guide.rb

```diff
@@ -9,6 +9,8 @@ class Guide < ActiveRecord::Base
   has_many :editions, dependent: :destroy
   has_many :topic_section_guides, autosave: true
 
+  has_one :topic, through: :topic_section_guides
```

Comment:
I failed to notice the existing method, I just knew I wanted a nice way of getting from Guide to its topic without having to traverse topic_section_guides. As it happens, the existing topic method does what I want so… that's all good then. Removed this and fixup-ed.